adcm_aio_client/core/objects/_accessors.py

```diff
@@ -62,13 +63,13 @@ async def list(self: Self) -> list[ReturnObject]:
         return [self._create_object(obj) for obj in results]
 
     async def _request_endpoint(self: Self, query: QueryParameters) -> RequesterResponse:
-        return await self._requester.get(*self._path, query=query)
+        return await self._requester.get(*self._path, query={**query, **self._filters})
```

Comment:
filters can't be passed directly to requester, they should come through conversion (simplification) first

see comment about naming filters, probably it should be just named differently

Reply:
renamed